src/components/TextInput/i18n/en.json

```diff
@@ -0,0 +1,3 @@
+{
+    "label_clear-button": "Clear input value"
```

Comment:
Just `Clear` wont be enough?

Reply:
I think it wont be clear enough what does it mean